test/spec/Component.spec.js

```diff
@@ -0,0 +1,16 @@
+import test from 'ava';
+import Component from '@/Component.svelte';
+
+test('validate component', t => {
+  const target = document.createElement('div');
+  const app = new Component({
+    target
+  });
+  t.snapshot(target);
+  t.is(typeof app.$$, 'object');
+  t.is(typeof app.$$.props, 'object');
+  t.is(typeof app.$$.on_mount, 'object');
+  t.is(typeof app.$$.on_destroy, 'object');
+  t.is(typeof app.$$.before_update, 'object');
+  t.is(typeof app.$$.after_update, 'object');
```

Comment:
I don't think there's any value in encouraging component authors to write test validating if the Svelte compiler works correctly.

Reply:
This was basically for example test, since the component was empty, the only way to show example tests was to check the hooks.

Reply:
I think this in itself would be a problem for a future PR. Demonstrating testing by writing an incorrect test isn't a very good demonstration.

It might be preferable to add just enough content to the example component that a test makes sense.